src/screens/UserPortal/Organizations/Organizations.tsx

```diff
@@ -363,7 +392,7 @@ export default function organizations(): React.JSX.Element {
 
   return (
     <>
-      {/* {hideDrawer ? (
+      {hideDrawer ? (
```

Comment:
```suggestion
      {/* hideDrawer ? (
```
This was part of the old sidebar toggle logic. I've commented it out for now since I've implemented a new one.